WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/variations/VariationsBulkUpdateAttrPickerViewModel.kt

```diff
@@ -43,9 +45,19 @@ class VariationsBulkUpdateAttrPickerViewModel @Inject constructor(
     init {
         val regularPriceValues = args.variationsToUpdate.asList().map { it.regularPrice }
         val salePriceValues = args.variationsToUpdate.asList().map { it.salePrice }
+
+        var stockQuantityValues: List<Double>? = null
```

Comment:
```suggestion
        val stockQuantityValues: List<Double> = args.variationsToUpdate.filter { it.isStockManaged }.map { it.stockQuantity }
```
🔍 Nitpick: This could just slightly simplify the code and the type would become non-nullable.